pkg/extensions/main.go

```diff
@@ -3,6 +3,7 @@ package extensions
 import (
 	_ "cloud.google.com/go/kms/apiv1"
 	_ "cloud.google.com/go/kms/apiv1/kmspb"
+	_ "cloud.google.com/go/spanner"
```

Comment:
does this have any (noticeable) effect on the OSS/enterprise distribution?

Reply:
I've posted PR https://github.com/grafana/grafana/pull/101341 to move imports under build tag.